examples/Benchmark.py

```diff
@@ -0,0 +1,310 @@
+    del tasks_nci, datasets_nci, transformers_nci
+    del train_dataset,valid_dataset,time1,time2,time3
+  
+  return None
+
+def benchmarkTrainAndValid(base_dir,train_dataset,valid_dataset,tasks,
+                           transformers,n_features = 1024,model = 'all',
+                           verbosity = 'high'):
+  train_scores = {}
+  valid_scores = {}
+  
+  # Initialize metrics
+  classification_metric = Metric(metrics.roc_auc_score, np.mean,
+                                 verbosity=verbosity,
+                                 mode="classification")
+  
+  assert model in ['all', 'tf', 'rf', 'keras']
+
+  if model == 'all' or model == 'tf':
+    # Initialize model folder
+    model_dir_tf = os.path.join(base_dir, "model_tf")
+    
+    # Building tensorflow MultiTaskDNN model
+    tensorflow_model = TensorflowMultiTaskClassifier(
+        len(tasks), n_features, model_dir_tf, dropouts=[.25],
+        learning_rate=0.001, weight_init_stddevs=[.1],
+        batch_size=64, verbosity=verbosity)
+    model_tf = TensorflowModel(tensorflow_model, model_dir_tf)
+ 
+    print('-------------------------------------')
+    print('Start fitting by tensorflow')
+    model_tf.fit(train_dataset)
+    train_evaluator = Evaluator(model_tf, train_dataset, transformers,
+                                verbosity=verbosity)
+    train_scores['tensorflow'] = train_evaluator.compute_model_performance(
+                                [classification_metric])['mean-roc_auc_score']
+    valid_evaluator = Evaluator(model_tf, valid_dataset, transformers,
+                                verbosity=verbosity)
+    valid_scores['tensorflow'] = valid_evaluator.compute_model_performance(
+                                [classification_metric])['mean-roc_auc_score']
+
+  
+  if model == 'all' or model == 'rf':
+    # Initialize model folder
+    model_dir_rf = os.path.join(base_dir, "model_rf")
+    
+    # Building scikit random forest model
+    def model_builder(model_dir_rf):
+      sklearn_model = RandomForestClassifier(
+        class_weight="balanced", n_estimators=500,n_jobs=-1)
+      return SklearnModel(sklearn_model, model_dir_rf)
+    model_rf = SingletaskToMultitask(tasks, model_builder, model_dir_rf)
+    
+    print('-------------------------------------')
+    print('Start fitting by random forest')
+    model_rf.fit(train_dataset)
+    train_evaluator = Evaluator(model_rf, train_dataset, transformers, 
+                                verbosity=verbosity)
+    train_scores['random_forest'] = train_evaluator.compute_model_performance(
+                                [classification_metric])['mean-roc_auc_score']
+    valid_evaluator = Evaluator(model_rf, valid_dataset, transformers, 
+                                verbosity=verbosity)
+    valid_scores['random_forest'] = valid_evaluator.compute_model_performance(
+                                [classification_metric])['mean-roc_auc_score']
+  
+  #currently not going to use this model
+  '''
```

Comment:
Can you delete this code? We can always add it in later if we want to use it again.